distribution/lib/Standard/Table/0.0.0-dev/src/Table.enso

```diff
@@ -3881,3 +3883,7 @@ make_fill_nothing_default_widget table cache=Nothing =
 ## PRIVATE
    Helper method for internal use to make a Table from a Java Table.
 from_java_table java_table = Table.Value java_table
+
+## PRIVATE
+Many_Files_List.from (that : Table) =
+    Read_Many_Helpers._find_files_list_in_table that
```

Comment:
I don't think this method should have an underscore. It is a public method of the Read_Many_Helpers module.

Reply:
Thanks, I wrote this before I fully understood where we are at with the `_` conventions and forgot to amend.